.travis.yml

```diff
@@ -26,6 +26,7 @@ install:
 
 script:
   - ./script/antlr4/setup.sh
+  - ./script/linker.sh
```

Comment:
I am not sure if we need to add this to the .travis file.

Reply:
So it is for a one-time run to link .githooks to .git/hooks/ folder. How should we go about it?

Reply:
My understanding is that since Githooks is more related to commits and Travis is used for incrementally building Eva, we might not need to add git related stuff here.

Reply:
Ok, I have re-edited it.